<launch>
	<node pkg="avr_bridge" type="bridge_node.py" name="motor_driver" 
				args="$(find navi_driver)/config/motor_driver.yaml">
		<remap from="cmd" to="drive/motor_cmd"/>
		<remap from="encoder" to="drive/encoder"/>
		<remap from="current" to="drive/current"/>
	</node>
	
	<node pkg="navi_driver" type="navi_driver.py" name="navi_driver">
		<rosparam commmand="load" file="$(find navi_driver)/diff_config.yaml" />
		<remap from="motor_cmd" to="drive/motor_cmd"/>
		<remap from="encoder" to="drive/encoder"/>
		<remap from="cmd_vel" to="drive/cmd_vel"/>
		<remap from="odom" to="drive/odom"/>
	</node>
</launch>
